packages/form/src/form-item.vue

```diff
@@ -265,8 +265,7 @@ export default defineComponent({
       const selfRules = props.rules
       const requiredRule =
         props.required !== undefined ? { required: !!props.required } : []
-
-      const prop = getPropByPath(formRules, props.prop || '', true)
+      const prop = getPropByPath(formRules, props.prop || '')
```

Comment:
I simply do not understand why do you have to change this, your modification based on `getPropByPath` is 
~~~diff
-export function getPropByPath(obj: any, path: string, strict: boolean): {
+export function getPropByPath(obj: any, path: string, strict?: boolean):
~~~
Tell me why you need to remove the parameter flag?

Reply:
also the doc, please run Delete or add form items dynamically demo and check the console